compiler/daml-lf-reader/src/DA/Daml/LF/Reader.hs

```diff
@@ -5,6 +5,7 @@ module DA.Daml.LF.Reader
     ( Manifest(..)
     , ManifestData(..)
     , manifestFromDar
+    , multiLineContent
```

Comment:
This was exposed just make testing easy, the other way was creating an archive which is very tedious :(